azure-monitor-opentelemetry/azure/monitor/opentelemetry/vendor/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -0,0 +1,661 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=too-many-locals
+
+"""
+The opentelemetry-instrumentation-asgi package provides an ASGI middleware that can be used
+on any ASGI framework (such as Django-channels / Quart) to track request timing through OpenTelemetry.
+
+Usage (Quart)
+-------------
+
+.. code-block:: python
+
+    from quart import Quart
+    from azure.monitor.opentelemetry.vendor.opentelemetry.instrumentation.asgi import OpenTelemetryMiddleware
```

Comment:
Do we even want to expose these doc strings? We don't expect the users to use these instrumentations directly do we?

Reply:
Removing